Volume/Supporting/UserData.swift

```diff
@@ -130,34 +131,40 @@ class UserData: ObservableObject {
         articleShoutoutsCounter[article.id, default: 0] += 1
     }
 
-    func set(article: Article, isSaved: Bool) {
+    func set(article: Article, isSaved: Bool, bookmarkRequestInProgress: Binding<Bool>) {
+        @Binding var requestInProgress: Bool
+        _requestInProgress = bookmarkRequestInProgress
```

Comment:
How come you create a new one here instead of using the one passed into the function? Are they not the same reference?

Reply:
so we don't have to set the wrapped value directly?